### ISSUE TITLE
1.20.5 raid farm borken.

### ISSUE THREAD

**Issue by NSTAG8R1** [misode/no-trial-chambers#26]
When I updated to 1.20.5 my raid farm doesnt work anymore. You cant get bad omen from killing a illager patrol leader.

**Comment by CyanNJ** [comment]
This is a 1.21 feature. 1.21 makes old raid farms not work, and bad omen doesn't apply through killing pillagers anymore. You get bad omen potions, however those are disabled in this datapack/mod. hope it helped.

**Comment by misode** [comment]
Thanks for bringing this to my attention. Unfortunately with a data pack I can't modify whether you get bad omen when killing a pillager captain, so I should probably just enable the bad omen potions in this data pack.

**Comment by NSTAG8R1** [comment]
> This is a 1.21 feature. 1.21 makes old raid farms not work, and bad omen doesn't apply through killing pillagers anymore. You get bad omen potions, however those are disabled in this datapack/mod. hope it helped.

Yes I am aware, thank you. It was just in 1.20.4 version of this data pack raid farms still worked but for some reason in the 1.20.5 version they stopped and went to the 1.21 version of how raid farms start.

**Comment by misode** [comment]
I've made pillagers drop ominous bottles and clarified in the readme that this also enables the new raid mechanics. If you don't want those, consider staying on 1.20.4 until the 1.21 release is finished.

**Comment by caitlynrw** [comment]
If you're able to install mods on the server or just in singleplayer, you can add the [Raid Restorer](https://modrinth.com/mod/raid-restore) mod and use the [1.3 version](https://github.com/misode/no-trial-chambers/releases/tag/v1.3) of the datapack for 1.20-type raids. I've tested this with [IanXO4's Stacking Raid Farm](https://youtu.be/n3mOlrMGjUg).

**Comment by devjta** [comment]
So its not possible, to simple have the auto-crafter without any other 1.21 changes (like raids or potions) in 1.20.5/6?

**Comment by misode** [comment]
> So its not possible, to simple have the auto-crafter without any other 1.21 changes (like raids or potions) in 1.20.5/6?

Correct, with data packs it's an all-or-nothing. You can enable the 1.21 experimental features or not.

**Comment by Helixjay** [comment]
was the datapack updated to allow pillagers to drop ominous bottles cause i havent gotten any from killing pillager captains

**Comment by misode** [comment]
It looks like I forgot to update the release on github when I fixed this, but you can get the 1.3.1 version on modrinth: https://modrinth.com/datapack/no-trial-chambers/version/1.3.1
